argus/backend/service/argus_service.py

```diff
@@ -370,6 +373,13 @@ def get_data_for_release_dashboard(self, release_name: str):
 
         return release, release_groups, release_tests
 
+    def get_distinct_release_versions(self, release_id: UUID | str) -> list[str]:
+        release_id = UUID(release_id) if isinstance(release_id, str) else release_id
+        rows = self.session.execute(self.scylla_versions_by_release, parameters=(release_id,))
+        unique_versions = {r["scylla_version"] for r in rows if r["scylla_version"]}
```

Comment:
nit: if row for some reason will not have field "scylla_version", here exception will be triggered. IMO it better to use r.get("scylla_version")?

Reply:
hmm, it should always be there, populated by `None` I think, due to how scylla-driver interface works. I might be wrong though.